src/aave-v3/SupplyVaultBase.sol

```diff
@@ -103,18 +109,28 @@ abstract contract SupplyVaultBase is ERC4626UpgradeableSafe, OwnableUpgradeable
             supplyBalance.inP2P.rayMul(morpho.p2pSupplyIndex(poolTokenMem));
     }
 
+    /// @notice Deposits an amount of assets into the vault and receive vault shares.
+    /// @param assets The amount of assets to deposit.
+    /// @param receiver The recipient of the vault shares.
     function deposit(uint256 assets, address receiver) public virtual override returns (uint256) {
         // Update the indexes to get the most up-to-date total assets balance.
         morpho.updateIndexes(poolToken);
         return super.deposit(assets, receiver);
     }
 
+    /// @notice Mints shares of the vault and transfers assets to the vault.
+    /// @param shares The number of shares to mint.
+    /// @param receiver The recipient of the vault shares.
```

Comment:
Can we just inherit doc from the ERC4626 implementation?

Reply:
But the version of ERC4626 we're using has broken comments (they have a typo in the reference)

Reply:
True...😭